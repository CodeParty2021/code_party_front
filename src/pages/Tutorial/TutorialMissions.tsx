import React from "react";
import { Link } from "react-router-dom";
type Props = {};

export const TutorialMissions: React.FC<Props> = () => {
  return (
    <div>
      <h1>チュートリアルミッション一覧のページ</h1>
      <Link to="/tutorial/world/:world_id/mission/:mission_id/step/:step_id">
        ミッションXのステップXに進む
      </Link>
      <Link to="/tutorial/world/:world_id/intro">
        このワールドのゲーム説明に進む
      </Link>
    </div>
  );
};
