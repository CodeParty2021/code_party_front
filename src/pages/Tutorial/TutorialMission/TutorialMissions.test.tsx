import React from "react";
import { shallow } from "enzyme";
import { TutorialMissions } from "./TutorialMissions";
describe("<TutorialMissions />", () => {
  it("auth snapshot test", () => {
    const wrapper = shallow(<TutorialMissions />);

    expect(wrapper.getElements()).toMatchSnapshot();
  });
});
