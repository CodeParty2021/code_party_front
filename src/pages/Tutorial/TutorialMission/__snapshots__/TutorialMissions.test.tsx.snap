// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TutorialMissions /> snapshot test 1`] = `
Array [
  <div>
    <h1>
      チュートリアルミッション一覧のページ
    </h1>
    <Link
      to="/tutorial/world/:world_id/mission/:mission_id/step/:step_id"
    >
      ミッションXのステップXに進む
    </Link>
    <Link
      to="/tutorial/world/:world_id/intro"
    >
      このワールドのゲーム説明に進む
    </Link>
  </div>,
]
`;
